lib/go-log/log.go

```diff
@@ -76,14 +77,14 @@ func logf(logger *log.Logger, format string, v ...interface{}) {
 	if logger == nil {
 		return
 	}
-	logger.Output(stackFrame, time.Now().UTC().Format(timeFormat)+": "+fmt.Sprintf(format, v...))
+	logger.Output(stackFrame, time.Now().UTC().Format(timeFormat)+": "+strings.Replace(strings.TrimSpace(fmt.Sprintf(format, v...)), "\n", `\n`, -1)+"\n")
 }
 
 func logln(logger *log.Logger, v ...interface{}) {
 	if logger == nil {
 		return
 	}
-	logger.Output(stackFrame, time.Now().UTC().Format(timeFormat)+": "+fmt.Sprintln(v...))
+	logger.Output(stackFrame, time.Now().UTC().Format(timeFormat)+": "+strings.Replace(strings.TrimSpace(fmt.Sprintln(v...)), "\n", `\n`, -1)+"\n")
```

Comment:
nit: these `+"\n"`s are unnecessary, because in a call to `log.Logger.Output`:

> A newline is appended if the last character of s is not already a newline.

Further nit: using `fmt.Sprintln` instead of `fmt.Sprintf` just gives `strings.TrimSpace` one extra character to delete.